apps/public-docsite/src/pages/Styles/FabricIconsPage/docs/web/FabricIconsUsage.md

```diff
@@ -1,4 +1,27 @@
-You can use Fluent UI's icons in a few ways, depending on if you're using Fluent UI React or Fabric Core.
+You can use Fluent UI's icons in a few ways, depending on if you're using React Icons, Fluent UI React or Fabric Core.
+
+### React Icons: SVG Based React Icons
+
+The Fluent UI SVG icon set is released under the MIT License.
+
+`@fluentui/react-icons` are SVG based icons wrapped in a React element. Because each icon is its own element, you only need to import what you need for your application.
```

Comment:
```suggestion
The `@fluentui/react-icons` package contains SVG-based icons wrapped in a React component. Because each icon is its own component, you only need to import and bundle the icons used in your application.
```